content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +83,95 @@ and has not been altered during transport.
 
 To enable DKIM, you must add a CNAME record to the DNS zone of your domain name:
 
-``odoo._domainkey IN CNAME odoo._domainkey.odoo.com.``
+`odoo._domainkey IN CNAME odoo._domainkey.odoo.com.`
 
 .. tip::
-   If your domain name is `mycompany.com`, you need to create a subdomain 
+   If your domain name is `mycompany.com`, you need to create a subdomain
    `odoo._domainkey.mycompany.com` whose canonical name is `odoo._domainkey.odoo.com.`.
 
 How to create or modify a CNAME record depends on the provider hosting the DNS zone of your domain
-name. The most common providers are list below.
+name. The most common providers are
+:ref:`listed below <email_communication/SPFDKIM_common_providers>`.
 
-You can check if your DKIM record is valid with a free tool like 
-`DKIM Core <https://dkimcore.org/tools/>`_. If a selector is asked, enter `odoo`.
+You can check if your DKIM record is valid with a free tool like `DKIM Core <https://dkimcore.org
+/tools/>`_. If a selector is asked, enter `odoo`.
 
 Check your DMARC policy
 =======================
 
-The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies SPF
-and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
+The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies
+SPF and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
 server what to do with an incoming email that fails the SPF and/or DKIM check.
 
 There are three DMARC policies:
-- ``p=none``
-- ``p=quarantine``
-- ``p=reject``
 
-``p=quarantine`` and ``p=reject`` instruct the server that receives an email to quarantine that
-email or ignore it if the SPF and/or DKIM check fails.
+- `p=none`
+- `p=quarantine`
+- `p=reject`
 
-If your domain name uses DMARC and has defined one of these policies, it is therefore imperative 
-to be SPF compliant or to enable DKIM.
+`p=quarantine` and `p=reject` instruct the server that receives an email to quarantine that email
+or ignore it if the SPF and/or DKIM check fails.
+
+If your domain name uses DMARC and has defined one of these policies, it is therefore imperative to
+be SPF compliant or to enable DKIM.
 
 .. danger::
-   Yahoo or AOL are examples of email providers with a DMARC policy set to ``p=reject``. We
-   strongly advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails
-   will never reach their recipient.
+   Yahoo or AOL are examples of email providers with a DMARC policy set to `p=reject`. We strongly
+   advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails will
```

Comment:
- Replace `We strongly advise` with `Odoo strongly advises`
- Replace `for your users` with `for the database users`